web/testing/issue005/header.js

```diff
@@ -0,0 +1,129 @@
+    The Toolbar (desktop browser) UI is best suited for allowing users to select 
+    the appropriate language and keyboard in this case.
+
+    Keyboards may also be specified by language name using addKeyboardsForLanguage()
+    for example:
+      keymanweb.addKeyboardsForLanguage('Burmese');
+    
+    Appending $ to the language name will again cause all available keyboards for that
+    language to be loaded rather than the default keyboard.
+        
+    The first call to addKeyboardsForLanguage() makes an additional call to the 
+    keyman API to load the current list of keyboard/language associations.
+
+    In this example, the following function loads the indicated keyboards,
+    and is called when the page loads. 
+*/
+
+  function loadKeyboards() 
+  { 
+    var kmw=window['keyman'] ? keyman : tavultesoft.keymanweb;
+    
+    // The first keyboard added will be the default keyboard for touch devices.
+    // For faster loading, it may be best for the default keybaord to be 
+    // locally sourced.
+    kmw.addKeyboards({id:'us',name:'English',languages:{id:'eng',name:'English'},
+      filename:'../us-1.0.js'});
+      
+    // Add more keyboards to the language menu, by keyboard name,
+    // keyboard name and language code, or just the ISO 639 language code.  
+    kmw.addKeyboards('french','european2@swe','european2@nor','@heb');
+  
+    // Add a keyboard by language name.  Note that the name must be spelled
+    // correctly, or the keyboard will not be found.  (Using ISO codes is
+    // usually easier.)
+    kmw.addKeyboardsForLanguage('Dzongkha');
+    
+    // Add a fully-specified, locally-sourced, keyboard with custom font  
+    kmw.addKeyboards({id:'lao_2008_basic',name:'Lao Basic',
+      languages:{
+        id:'lao',name:'Lao',region:'Asia',
+        font:{family:'LaoWeb',source:['../font/saysettha_web.ttf','../font/saysettha_web.woff','../font/saysettha_web.eot']}
+        },
+      filename:'../lao_2008_basic.js'
+      });   
+
+    // The following two optional calls should be delayed until language menus are fully loaded:
+    //  (a) a specific mapped input element input is focused, to ensure that the OSK appears
+    //  (b) a specific keyboard is loaded, rather than the keyboard last used.         
+  //window.setTimeout(function(){kmw.setActiveElement('ta1',true);},2500);
+  //window.setTimeout(function(){kmw.setActiveKeyboard('Keyboard_french','fra');},3000);
+  
+    // Note that locally specified keyboards will be listed before keyboards 
+    // requested from the remote server by user interfaces that do not order
+    // keyboards alphabetically by language.
+  }
+  
+    // Script to allow a user to add any keyboard to the keyboard menu 
+  function addKeyboard(n)
+  { 
+    var sKbd, kmw=window['keyman'] ? keyman : tavultesoft.keymanweb;
+    switch(n)
+    {
+      case 1:
+        sKbd=document.getElementById('kbd_id1').value;
+        kmw.addKeyboards(sKbd);
+        break;
+      case 2:
```

Comment:
It seems your test page only uses `addKeyboard(1)`
Were you planning this `switch` statement, or can it be removed?

Reply:
It can be; that code is lifted straight from the original uncompiled.html sample page.  In case we decide later to reinstate the code, I left it in place.